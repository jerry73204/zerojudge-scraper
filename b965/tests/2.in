7 7 4
8 3 3 4 4 6 0
9 4 0 2 0 4 9
6 5 7 8 5 9 4
7 5 2 8 5 4 1
9 4 4 7 9 5 1
2 9 5 9 0 7 4
1 5 9 5 5 3 7
1 1 0 1