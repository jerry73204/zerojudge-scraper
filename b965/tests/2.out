7 7
7 4 1 1 4 9 0
3 7 5 4 9 4 6
5 0 9 5 5 0 4
5 9 7 8 8 2 4
9 5 4 2 7 0 3
5 9 4 5 5 4 3
1 2 9 7 6 9 8